md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ServersAPI.java

```diff
@@ -69,12 +68,12 @@ RestWrapper get(
                 server.setServerMetaInfo(jpaServers.getServerMetainfo());
                 server.setSshPrivateKey(jpaServers.getSshPrivateKey());
             }
-            //  server = s.selectOne("call_procedures.GetServers", server);
 
             restWrapper = new RestWrapper(server, RestWrapper.OK);
             LOGGER.info("Record with ID:" + serverId + " selected from Servers by User:" + principal.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Record with ID:" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)